### PR TITLE
docs: 📝 add section on team calendars

### PR DESCRIPTION
**Pull request by signekb**
## Description

These changes are done because I kept forgetting how to do this ☺️ 

## Reviewer Focus

This PR needs a quick review.

## Checklist

- [X] Ran spell-check
- [X] Formatted Markdown
- [X] Rendered website locally

### CODE REVIEW

File: onboarding/index.qmd [hunk 1]
```diff
@@ -6,42 +6,124 @@ title: Onboarding
...
+    -   Talk to the Team Leader about which IT equipment you will need
+        (e.g., computer, mouse, keyboard, and dock station) and Team
+        Leader will help you with ordering it.
+
+## Team calendars
+
+We have a set of team calendars to schedule meetings, annual leave,
+iterations, and so on. For this, we use [Team Up](https://teamup.com/)
+calendars. Someone from the team will send you an invite link. You will
+need to create an account on teamup.com for this.
```
Comment (lwjohnst86) [inline onboarding/index.qmd:40]:
```suggestion
need to create an account on Teamup for this.
```

File: onboarding/index.qmd [hunk 2]
```diff
@@ -6,42 +6,124 @@ title: Onboarding
...
+        access the Seedcase repositories).
+3.  **Order IT equipment**
+    -   Talk to the Team Leader about which IT equipment you will need
+        (e.g., computer, mouse, keyboard, and dock station) and Team
+        Leader will help you with ordering it.
+
+## Team calendars
+
+We have a set of team calendars to schedule meetings, annual leave,
+iterations, and so on. For this, we use [Team Up](https://teamup.com/)
```
Comment (lwjohnst86) [inline onboarding/index.qmd:38]:
```suggestion
iterations, and so on. For this, we use [Teamup](https://teamup.com/)
```

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Nice! Only some small suggestions :relaxed:

**Review by lwjohnst86 — APPROVED** [review]

**Review by K-Beicher — COMMENTED** [review]
That looks like it will work on a Mac.

**Pull request merged**